python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -274,21 +274,20 @@ def get_parameters(self):
         parent_block = self.parent_block()
 
         local_inputs = set()
-
```

Comment:
The previous logic cannot calculate parameters that used and updated by the same operator.